docs/blockchain-integrations-framework/index.md

```diff
@@ -0,0 +1,229 @@
+        [tests.v1-7-0.overrides.shorter-test]
+        test_duration=2 # minutes
+
+    [tests.v1-9-0]
+
+        [tests.v1-9-0.base]
+        keep_environments="Never" # Always | OnFail | Never
+        # Image repo to pull the Chainlink image from
+        chainlink_image="public.ecr.aws/chainlink/chainlink"
+        # Version of the Chainlink image to pull
+        chainlink_version="1.9.0"
+        # Name of the person running the tests (no spaces)
+        chainlink_env_user="My-Name"
+        test_log_level="info" # info | debug | trace
+        node_count=6  
+        test_duration=15 # minutes
+        contract_count=2
+        node_funding=0.1 # ETH
+        round_timeout=15 # minutes
+        expected_round_time=2 # minutes
+        time_between_rounds=1 # minutes
+
+        [tests.v1-9-0.overrides.shorter-test]
+        test_duration=2 # minutes
+```
+
+If you completed the full [steps to set up your environment](#set-up-your-environment), you can run a soak test using the following steps:
+
+1. Create a file called `example.toml` and paste the example TOML file above.
+1. Edit the file to add your `private_keys` under the `networks.evm.base` config.
+1. Modify the `ws_urls` under `networks.evm.overrides.goerli` with your specified RPC endpoint URL. Optionally, you can create additional overrides for different networks`.
+1. Start running a soak test `bif integration soak ./example.toml`.
+1. The interactive menu opens. Select evm, your desired network overrides, the Chainlink node version, and your test length settings.
+1. While your test is running, view your local Grafana instance at `localhost:3000` to see test metrics.
+1. Check node logs: `bif integration logs NAMESPACE [--level LOG_LEVEL]`
+
+As you test additional networks, modify the TOML file and configure it with the details for your network.
+
+### Troubleshooting integration tests
+
+There are some common issues with soak testing that you can troubleshoot.
+
+**Pods remaining from failed tests:**
+
+If a soak test fails, the pods used for testing often remain running. Use `kubectl` to identify and delete these pods:
+
+1. List all pods by their namespaces: `kubectl get pods --all-namespaces`
+1. Delete the deployments that you no longer need by namespace: `kubectl delete --all deployments --namespace=ocr-soak-goerli-50b0c`
+1. Modify your test configuration and run `bif integration soak ./example.toml` again.
+
+**Insufficient resources:**
+
+Your system must meet the defined [Requirements](#requirements). Running on systems without sufficient resources will cause soak tests to fail because some containers for Chainlink nodes will not be able to start.
+
+- If you are running on a virtual machine, add additional vCPUs, system memory, or storage.
+- Run your soak tests with fewer nodes. The default value in the TOML config is `node_count=6`, which is recommended for a thorough soak test. You can adjust this value if you are unable to obtain the necessary system resources.
```

Comment:
Node count is no longer set through TOML. It can be set via `bif infra cluster create --count 6` or `bif integration test --count 6`.
```suggestion
- Run your soak tests with fewer nodes. The default node count is 6 per cluster, which is recommended for a thorough soak test. You can adjust this value if you are unable to obtain the necessary system resources.
```